### PR TITLE
Add an api endpoint for making payments

### PR DESCRIPTION
**Pull request by justinvdm**
The plan is to use [stellar-lib](https://github.com/stellar/stellar-lib) for this, though we will probably have to change to a client library that talks to their 'horizon' service, once that exists.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review. Created #4 for the only TODO I saw not covered by existing tickets.

File: tests/fixtures/requests.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+module.exports = fixtureSet({
+  setup: function(fixtures) {
+    fixtures
+      .add('stellarApi:url', 'https://test.stellar.org:9002')
+
+      .add('stellarApi:issuer', 'gBAde4mkDijZatAdNhBzCsuC7GP4MzhA3B')
+
+      .add('stellarApi:payment:str', nock(fixtures('stellarApi:url'))
+        .post('/', {"method":"submit","params":[{"secret":"sfSU4rF2UuKTCQG5kTukEqNsuUgUTgf2UuC2gsEokxtypwCeJbZ","tx_json":{"TransactionType":"Payment","Account":"gajc65uAsK4AgQM771SqkntWyVQL48GBjK","Destination":"gLk2FP9PFw1TFAt7kyAVuJpr4vLapiVRSJ","Amount":1}}]})
```
Comment (hodgestar) [inline tests/fixtures/requests.js:18]: Just double-checking that the secret used doesn't belong to a real account.

**Comment by hodgestar** [comment]
:+1: if the answer to the secret question is sane. :)

File: tests/fixtures/requests.js (hunk 1 above)
Comment (justinvdm) [inline tests/fixtures/requests.js:18]: The secret belongs to a test account I created on the old stellar test network. I asked the friendly robot at https://www.stellar.org/ to give me some credits. So this is real in that the address exists in the test stellar network's ledger, but that's pretty much it (no real currency or personal information that can be lost or anything).

Comment (hodgestar) [inline tests/fixtures/requests.js:18]: Woot. Sounds good. :+1:

**Pull request merged**